### PR TITLE
Add possibility to run application on docker.

### PR DESCRIPTION
**Pull request by makaroshyna**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+services:
+  mysqldb:
+    platform: linux/x86_64
+    image: mysql
+    restart: no
+    env_file: ./.env
+    environment:
+      MYSQL_DATABASE: ${MYSQLDB_DATABASE}
+      MYSQL_USER: ${MYSQLDB_USER}
+      MYSQL_PASSWORD: ${MYSQLDB_PASSWORD}
```
Comment (nklimovych) [inline docker-compose.yml:12]: Is it really necessary to have these variables?

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    build: .
+    env_file: ./.env
+    ports:
+      - ${SPRING_LOCAL_PORT}:${SPRING_DOCKER_PORT}
+      - ${DEBUG_PORT}:${DEBUG_PORT}
+    environment:
+      SPRING_APPLICATION_JSON: '{
+        "spring.datasource.url": "jdbc:mysql://mysqldb:${MYSQLDB_DOCKER_PORT}/${MYSQLDB_DATABASE}?serverTimezone=UTC", 
+        "spring.datasource.username": "${MYSQLDB_USER}", 
+        "spring.datasource.password": "${MYSQLDB_PASSWORD}"
```
Comment (nklimovych) [inline docker-compose.yml:35]: It is more question than suggestion, but in my case, it doesn't work without setting `jwt.secret`:
```suggestion
        "spring.datasource.password": "${MYSQLDB_PASSWORD}"
        "jwt.secret": "${JWT_SECRET}"
```

**Review by nklimovych — APPROVED** [review]
Looking Great!

File: docker-compose.yml (hunk 1 above)
Comment (makaroshyna) [reply docker-compose.yml:12]: No, it's not necessary to have a separate user for the database. I used that, but it's optional :)

File: docker-compose.yml (hunk 2 above)
Comment (makaroshyna) [reply docker-compose.yml:35]: Mine worked without jwt.secret, but at one point I thought about adding it here.

**Review by andrii-hoienko — APPROVED** [review]
Good job!

**Pull request merged**